shell/platform/darwin/macos/framework/Headers/FlutterPluginRegistrarMacOS.h

```diff
@@ -36,11 +36,16 @@ FLUTTER_DARWIN_EXPORT
 @property(nonnull, readonly) id<FlutterTextureRegistry> textures;
 
 /**
- * The default view displaying Flutter content.
+ * The view displaying Flutter content.
  *
- * This method may return |nil|, for instance in a headless environment.
+ * Some single-view APIs will eventually be replaced by their multi-view
+ * variant. During the deprecation period, the single-view APIs will coexist
+ * with and work with the multi-view APIs as if the other views don't exist.
+ * For backward compatibility, single-view APIs will always operate on this
+ * view, which is the first view assigned to the engine.
```

Comment:
> which is the first view assigned to the engine.

This property will be nil if you don't link a view controller to the implicit view.

Reply:
> For backward compatibility, single-view APIs will always operate on this view

This conflicts with the "will eventually be replaced" comment. Which single-view APIs do you have in mind here? 1P plugins will move off this API.